client/lbclient/datasources.local.js

```diff
@@ -2,6 +2,6 @@ var GLOBAL_CONFIG = require('../../global-config');
 
 module.exports = {
   remote: {
-    url: GLOBAL_CONFIG.restApiUrl
+    url: 'http://localhost:3000' + GLOBAL_CONFIG.restApiUrl
```

Comment:
I figured out the problem. This might be a bug, but you need to add the FULL URL for things to work properly. Thoughts?